src/main/frege/frege/nativegen/Main.fr

```diff
@@ -170,14 +175,14 @@ parseArgs args = go optDefault args where
   go optSoFar ("-p" : packageName : rest)
     = go optSoFar.{name = packageName, category = Package} rest
   go optSoFar ("-c" : className : rest)
-    = go optSoFar.{name = className, category = Class} rest
+    = go optSoFar.{name = optSoFar.name ++ (if optSoFar.name == "" then "" else ",") ++ className, category = Class} rest
   go optSoFar ("-d" : directory : rest)
     = go optSoFar.{outputDir = directory} rest
   go optSoFar ("-t" : typesFilePath : rest)
     = go optSoFar.{typesFilePath = typesFilePath} rest
   go _ (invalidOption:_) = Left $ "Invalid option: " ++ invalidOption
 
-data Category = Package | Class
+data Category = Package | Class | Types
```

Comment:
Hint: I improved the Class type too. You can use it like that:

$ frege-native-gen -c one.class -c another.class -c so.many.classes -c you.want ...